CONTRIBUTING.md

```diff
@@ -2,55 +2,69 @@
+* Make sure to read and follow [code conventions](docs/code_conventions).
+* Run and verify that the various linters we support pass as detailed [here](docs/linting).
+* For Scripts/Integrations make sure to create unit tests as documented [here](docs/tests/unit-testing)
+* Create a test playbook as documented [here](docs/tests). **Note**: for small Scripts that have unit tests, a test playbook is optional.
+* Validate that our validation hooks pass. If you used `.hooks/bootstrap` as documented in [Getting Started](docs/getting_started) the validation hook will run automatically upon commit. You can also run the validation hooks manually by running `.hooks/pre-commit`.
+* Document you integration a detailed [here](docs/integration_documentation).
+* Document your changes in the relevant changelog file as detailed [here](docs/release_notes).
+  
  - Circle CI build must be green [here](https://github.com/demisto/content/tree/master/docs/tests/circleci)
 
-You should now be ready to push your changes to the Content GitHub repository, please do as follows.
+You should now be ready to create a pull request.
 
-## Push changes to GitHub
+## Create a Pull Request on GitHub
 
-Demisto content is MIT Licensed and accepts contributions via GitHub pull requests.
 If you are a first time GitHub contributor, please look at these links explaining on how to create a Pull Request to a GitHub repo:
 * https://guides.github.com/activities/forking/
 * https://help.github.com/articles/creating-a-pull-request-from-a-fork/
 
 **Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
 
+## Review Process
+Once you open a pull request we have a bot (`@content-bot`) that will assist you and provide comments and reminders about the state of the pull request. Additionally, a member of the team will be assigned to review the pull request. Comments will be provided by the team member as the review process progresses.
+
+You will see a few [GitHub Status Checks](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks) that help validate that your pull request is according to our standards:
+
+* **ci/circleci: build**: We use [CircleCI](https://circleci.com/gh/demisto/content) to run a full build on each commit of your pull request. The build will run our content validation hooks, linting and unit test. We require that the build pass (green build). If it is failing our content-bot will also add instructive comments about what sections are failing in the build. Follow the `details` link of the status to see the full build UI of CircleCI. More details about our build flow of CircleCI is available [here](docs/tests/circleci).
```

Comment:
```suggestion
* **CI build**: We use [CircleCI](https://circleci.com/gh/demisto/content) to run a full build on each commit of your pull request. The build will run our content validation hooks, linting and unit test. We require that the build pass (green build). If it is failing our content-bot will also add instructive comments about what sections are failing in the build. Follow the `details` link of the status to see the full build UI of CircleCI. More details about our build flow of CircleCI is available [here](docs/tests/circleci).
```

Reply:
I want to use the same names we show at status checks in Github

Reply:
👍